layers/+lang/reason/packages.el

```diff
@@ -0,0 +1,160 @@
+;;; packages.el --- reason layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Fredrik Dyrkell
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst reason-packages
+  '(company
```

Comment:
Sort packages alphabetically.

And the functions should probably be sorted alphabetically based on the package name.

Possibly like this:
#### before:
(defun reason/post-init-company ()
(defun reason/init-reason-mode ()
(defun reason/post-init-evil-matchit ()
(defun reason/pre-init-popwin ()
(defun reason/pre-init-utop ()
(defun reason/post-init-merlin ()
(defun flycheck-ocaml-reason-setup ()
(defun reason/post-init-flycheck ()
(defun reason/post-init-flycheck-ocaml ()
(defun ocaml/init-flycheck-ocaml ()


#### after:
(defun reason/post-init-company ()
(defun reason/post-init-evil-matchit ()
(defun reason/post-init-flycheck ()
(defun reason/post-init-flycheck-ocaml ()
(defun ocaml/init-flycheck-ocaml ()
(defun flycheck-ocaml-reason-setup ()
(defun reason/post-init-merlin ()
(defun reason/pre-init-popwin ()
(defun reason/init-reason-mode ()
(defun reason/pre-init-utop ()